ntbs-service/DataMigration/Exceptions/MarkingNotificationsAsImportedFailedException.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ntbs_service.Models.Entities;
+
+namespace ntbs_service.DataMigration.Exceptions
+{
+    public class MarkingNotificationsAsImportedFailedException : Exception
+    {
+        public MarkingNotificationsAsImportedFailedException(ICollection<Notification> notifications, Exception cause)
+            : base(ErrorMessage(notifications), cause)
+        {
+        }
+
+        private static string ErrorMessage(ICollection<Notification> notifications)
+        {
+            var ids = notifications.Select(n => n.NotificationId);
+            return $"Failed to mark notifications {ids} as imported in the migration db. " +
```

Comment:
Did you confirm this string interpolation did what you expected?
I would assume that an IEnumerable<int> doesn't print this nicely.

Reply:
Nope, you're completely right and I just rushed through this!